src/pal/include/occlum_pal_api.h

```diff
@@ -93,6 +93,13 @@ struct occlum_pal_create_process_args {
     int *pid;
 };
 
+// todo: more detailed description
+struct host_file_buffer {
+    const char *resolv_conf_ptr;
```

Comment:
I think `_ptr` is not accurate, maybe we can use `resolv_conf_buf` `hosts_buf` `hostname_buf`

Reply:
Done